AUTHORS

```diff
@@ -0,0 +1,3 @@
+Thomas Oster
+
+Google Inc.
```

Comment:
I work at Google, and our policies for contributing to open source software include adding this reference.